var/spack/repos/builtin/packages/py-reprozip/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyReprozip(PythonPackage):
+    """A tool simplifying creating reproducible experiments from CLI executions.
+
+    It tracks operating system calls and creates a bundle that contains all the
+    binaries, files and dependencies required to run a given command on the
+    author’s computational environment (packing step). A reviewer can then
+    extract the experiment in his environment to reproduce the results
+    (unpacking step).
+
+    This is the component responsible for the packing step on Linux
+    distributions.
+
+    Please refer to reprounzip, reprounzip-vagrant, and reprounzip-docker for
+    other components and plugins.
+
+    """
+
+    homepage = "https://www.reprozip.org/"
+
+    pypi = "reprozip/reprozip-1.2.tar.gz"
+
+    maintainers("charmoniumQ")
+
+    version("1.2", md5="a98b7f04c52c60072e3c42da21997d3ad41161ff6cb1139e18cda8d3012120f9")
+
+    def setup_build_environment(self, env) -> None:
+        env.set("CFLAGS", " -I%s/include" % self.spec["sqlite"].prefix)
```

Comment:
Are Spack's compiler wrappers not working? This should be added automatically.

Reply:
Yeah, I couldn't get it to build without a flag like this or what I used to have:

```
    def global_options(self, spec, prefix):
        return ["build_ext", "-I%s/include" % self.spec["sqlite"].prefix]
```

Reply:
@haampie @scheibelp @skosukhin any idea why Spack's compiler wrappers aren't being used here? Maybe they don't work correctly for Python?